server/chat-plugins/scavenger-games.ts

```diff
@@ -213,6 +220,46 @@ const TWISTS: {[k: string]: Twist} = {
 		},
 	},
 
+	spamfilter: {
+		name: 'Spam Filter',
+		id: 'spamfilter',
+
+		desc: 'Every wrong answer adds 30 seconds to your final time!',
+
+		onIncorrectAnswer(player: ScavengerHuntPlayer, value: string) {
+			if (!player.incorrect) player.incorrect = [];
+			const id = `${player.currentQuestion}-${value}`;
+			if (player.incorrect.includes(id)) return;
+
+			player.incorrect.push(id);
+		},
+
+		onComplete(player, time, blitz) {
+			const seconds = toSeconds(time);
+			if (!player.incorrect) return {name: player.name, total: seconds, blitz, time, original_time: time};
+
+			const deduction = 30 * player.incorrect.length;
+			const total = seconds + deduction;
```

Comment:
```suggestion
			const total = seconds + (30 * player.incorrect.length);
```
Nitpick. Declaring variables you only use once is messier, imo.